jobs/build/send-umb-messages/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env groovy
+
+node {
+    properties([
+        disableConcurrentBuilds(),
+    ])
+
+    // we only care to publish messages for the following releases
+    releases = ["4-stable", "4.2.0-0.nightly"]
+
+    stage("send UMB messages for new releases") {
+        dir ("~/.cache/releases") {
```

Comment:
As long as this is not in the workspace it will work, but I'm not sure where on the Jenkins host we should put it.

Reply:
This will work for now. We can change it if we need to.

Reply:
I went ahead and created this on the host.